graveler/committed/apply.go

```diff
@@ -98,6 +110,12 @@ func Apply(ctx context.Context, writer MetaRangeWriter, source Iterator, diffs g
 	haveSource, haveDiffs := source.Next(), diffs.Next()
 	changed := false
 	for haveSource && haveDiffs {
+		select {
+		case <-ctx.Done():
+			return ret, ctx.Err()
```

Comment:
How was this tested? (question applies to all other places where this type of check has been introduced..)